ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/CommandParser.java

```diff
@@ -350,11 +386,54 @@ public String getSourceName() {
   }
 
   /**
-   * Represents commands that deal with connectors.
+   * Represents ksqlDB CREATE CONNECTOR statements.
    */
-  public static class SqlConnectorStatement extends SqlCommand {
-    SqlConnectorStatement(final String command) {
+  public static class SqlCreateConnectorStatement extends SqlCommand {
+    final String name;
+    final boolean isSource;
+    final Map<String, Object> properties;
+
+    SqlCreateConnectorStatement(
+        final String command,
+        final String name,
+        final boolean isSource,
+        final Map<String, Object> properties
+    ) {
       super(command);
+      this.name = name;
+      this.isSource = isSource;
+      this.properties = properties;
```

Comment:
nit: validate that this is non-null, and similarly for the other params and SqlDropConnectorStatement below.

As an FYI, Java primitive types (including boolean) don't need the validation since they may not be null. (I notice that `SqlPropertyCommand` has a non-null check for the boolean `set` param but not the `Optional` `value` param -- those should be flipped.)